test/behaviour/config/Parameters.java

```diff
@@ -130,7 +130,8 @@ public List<Arguments.Transaction.Type> transaction_types(List<String> values) {
     public enum RootLabel {
         ENTITY("entity"),
         ATTRIBUTE("attribute"),
-        RELATION("relation");
+        RELATION("relation"),
+        THING("thing");
```

Comment:
Added solely for us to be able to do `Then delete thing type: thing; throws exception` in BDD tests.